src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelDeletionTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.modifications;
+
+import com.powsybl.iidm.network.IdentifiableType;
+import com.powsybl.iidm.network.Network;
+import lombok.SneakyThrows;
+import org.gridsuite.modification.server.dto.EquipmentDeletionInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.utils.MatcherEquipmentDeletionInfos;
+import org.gridsuite.modification.server.utils.NetworkCreation;
+import org.junit.Test;
+import org.springframework.http.MediaType;
+
+import java.util.UUID;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+public class VoltageLevelDeletionTest extends AbstractNetworkModificationTest {
+    @Override
+    protected Network createNetwork(UUID networkUuid) {
+        return NetworkCreation.create(networkUuid, true);
+    }
+
+    @Override
+    protected ModificationInfos buildModification() {
+        return EquipmentDeletionInfos.builder()
+                .equipmentType("VOLTAGE_LEVEL")
+                .equipmentId("v1")
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return EquipmentDeletionInfos.builder()
+                .equipmentType("VOLTAGE_LEVEL")
+                .equipmentId("v1")
+                .build();
+    }
+
+    @Override
+    protected MatcherEquipmentDeletionInfos createMatcher(ModificationInfos modificationInfos) {
+        return MatcherEquipmentDeletionInfos.createMatcherEquipmentDeletionInfos((EquipmentDeletionInfos) modificationInfos);
+    }
+
+    @Override
+    protected void assertNetworkAfterCreation() {
+        assertNull(getNetwork().getVoltageLevel("v1"));
+    }
+
+    @Override
+    protected void assertNetworkAfterDeletion() {
+        assertNotNull(getNetwork().getVoltageLevel("v1"));
+    }
+
+    @SneakyThrows
+    @Test
+    public void testRemoveVoltageLevel() {
```

Comment:
You can just move all the assertions in this test to assertNetworkAfterCreation method, and then remove this test method ...

Reply:
done